platform/src/main/java/org/stellar/anchor/platform/payment/observer/circle/CirclePaymentObserverService.java

```diff
@@ -91,11 +92,16 @@ public void handleCircleNotification(CircleNotification circleNotification)
   public void handleSubscriptionConfirmationNotification(CircleNotification circleNotification)
       throws BadRequestException {
     String subscribeUrl = circleNotification.getSubscribeUrl();
-    if (subscribeUrl == null) {
+    if (StringHelper.isEmpty(subscribeUrl)) {
```

Comment:
💡 Can we add static import?

```suggestion
    if (isEmpty(subscribeUrl)) {
```

Reply:
I've adopted your suggestion but I'm not a big fan of static imports unless the methods are widely used.

I also think `StringHelper` provides useful context to the function call, while a plain `isEmpty` can be mistaken by a local method.